### PR TITLE
publish snapshots to Gluon nexus repository

### LINKED ISSUE #8: Add workflow to release snapshots to Gluon private Maven repo

### PR DESCRIPTION
**Pull request by abhinayagarwal**
Fixes #8

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -97,4 +97,51 @@
...
+            <email>eugene.ryzhikov @gluonhq.com</email>
+            <organization>Gluon</organization>
+            <organizationUrl>https://www.gluonhq.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>José Pereda</name>
+            <email>jose.pereda@gluonhq.com</email>
+            <organization>Gluon</organization>
+            <organizationUrl>https://www.gluonhq.com</organizationUrl>
+        </developer>
```
[the remark this answers is not in the document]
Comment (abhinayagarwal) [reply pom.xml:113]: we can always append this list :)

[the remark this answers is not in the document]
Comment (jperedadnr) [reply pom.xml:113]: Right, this is one of the things that once created stays as is forever ;)

File: pom.xml [hunk 2]
```diff
@@ -97,4 +97,51 @@
             </properties>
         </profile>
     </profiles>
+
+    <developers>
+        <developer>
+            <name>Eugene Ryzhikov</name>
+            <email>eugene.ryzhikov @gluonhq.com</email>
```
Comment (eugener) [inline pom.xml:104]: Remove space in my email address

File: pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (eugener) [reply pom.xml:113]: Agree! More developers have to be added here 😎

**Review by eugener — CHANGES_REQUESTED** [review]

**Pull request merged**